### ISSUE TITLE
Unique array error

### ISSUE THREAD

**Issue by YandiBanyuKarimaWaly** [Brendonovich/prisma-client-rust#417]
I come from Javascript background so apologize in advance if I don't use the correct term/syntax. I am making a tree data structure using materialized path. Each path in a database should be a unique array. However, when I generate the client it created an invalid client.

Here is an example model:

```prisma
model item {
    numeric_id BigInt   @unique @default(autoincrement())
    path       BigInt[] @unique
}
```

Here is the offending snippet in the generated client:

```Rust
impl From<UniqueWhereParam> for WhereParam {
    fn from(value: UniqueWhereParam) -> Self {
        match value {
            UniqueWhereParam::NumericIdEquals(value) => {
                Self::NumericId(_prisma::read_filters::BigIntFilter::Equals(value))
            }
            UniqueWhereParam::PathEquals(value) => {
                Self::Path(_prisma::read_filters::BigIntListFilter::Equals(value))
            }
        }
    }
}
```

The problem is that `value` on `PathEquals` is `i64` and the expected type for `BigIntListFilter::Equals` is `Vec<i64>`.

EDIT : Forgot to mention, my current workaround is manually editing the enum. I haven't tested it but it compiles just fine.

Before:

```Rust
pub enum UniqueWhereParam {
    NumericIdEquals(i64),
    PathEquals(i64),
}
```

After:

```Rust
pub enum UniqueWhereParam {
    NumericIdEquals(i64),
    PathEquals(Vec<i64>),
}
```

**Comment by RubberDuckShobe** [comment]
Facing the same issue, this is the only thing preventing me from using this.

The model:
```prisma
model Song {
    id Int @id @default(autoincrement())

    scores Score[]
    shouts Shout[]

    title             String
    artist            String
    tags              String[] @default([])
    mbid              String?
    musicbrainzTitle  String?
    musicbrainzArtist String?
    musicbrainzLength Int?
    mistagLock        Boolean  @default(false)
    coverUrl          String?
    smallCoverUrl     String?

    @@unique([title, artist, tags])
}
```

The error:
```
mismatched types
expected struct `std::string::String`
   found struct `std::vec::Vec<std::string::String>`
```

Generated client:
```rust
Self::TitleArtistTagsEquals(title, artist, tags) => (
    "title_artist_tags",
    ::prisma_client_rust::SerializedWhereValue::Object(vec![
        (
            title::NAME.to_string(),
            ::prisma_client_rust::PrismaValue::String(title),
        ),
        (
            artist::NAME.to_string(),
            ::prisma_client_rust::PrismaValue::String(artist),
        ),
        (
            tags::NAME.to_string(),
            ::prisma_client_rust::PrismaValue::String(tags),
        ),
    ]),
),
```

**Comment by Brendonovich** [comment]
Fixed in 33e5ae1
